Dockerfile_browser_tools.dockerfile

```diff
@@ -0,0 +1,18 @@
+# Build browser tools for test-executor
+# Use the cimg/ruby:3.2.2-browsers image as the base image to extend out
+FROM cimg/ruby:3.2.2-browsers
+
+WORKDIR /app
+
+# Install Chrome
+RUN npx puppeteer browsers install chrome
+
+# Install Puppeteer with Chromium
+RUN sudo yarn add puppeteer@21.5.0
+
+# Install PDFTK
+RUN sudo add-apt-repository --yes ppa:malteworld/ppa || true
+RUN sudo apt update --allow-unauthenticated || true
+RUN sudo apt install pdftk --allow-unauthenticated || true
```

Comment:
Like wise here we can put them into the same RUN command like so: 

```
RUN sudo add-apt-repository --yes ppa:malteworld/ppa \
    && sudo apt-get update --allow-unauthenticated \
    && sudo apt-get install -y pdftk --allow-unauthenticated
```

Just a question though, don't we want things to fail so we know there is an issue with any of the commands if we use `|| true`?

Reply:
@willc-work - Also agree with getting rid of the `|| true`, I should have removed this now that we have trialed and tested this.